### PR TITLE
SVG downloads, see also line

### PR DESCRIPTION
**Pull request by jo-pol**
Just one example as a suggestion for the rest

* the "see also" line outside the table
* a single SVG link clutters the table less

One SVG download per family is not enough. Shifting pins not always allows to morph each family member into each other member.

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -40,12 +40,13 @@ Traditional Grounds
 |     |     |     |     |     |
 |:---|:---|:---|:---|:---|
 [![](tl/4/2x2_3.png)][2x2_3] | [![](tl/4/1x1_1.png)][1x1_1] | [![](tl/4/1x2_1.png)][1x2_1] | [![](tl/4/1x3_1.png)][1x3_1] | [![](tl/4/1x4_2.png)][1x4_2]
-<a href="tl/4/2x2_3.txt" download="2x2_3.txt">2x2_3</a> <br> <a href="/GroundForge/sheet.html?{{ "patch=5-,-5;checker" }}">SVG</a> | <a href="tl/4/1x1_1.txt" download="1x1_1.txt">1x1_1</a> <br> <a href="/GroundForge/sheet.html?{{ "patch=6;checker" }}">SVG</a> | <a href="tl/4/1x2_1.txt" download="1x2_1.txt">1x2_1</a> <br> <a href="/GroundForge/sheet.html?{{ "patch=53;checker" }}">SVG</a> | <a href="tl/4/1x3_1.txt" download="1x3_1.txt">1x3_1</a> <br> <a href="/GroundForge/sheet.html?{{ "patch=563;checker" }}">SVG</a> | <a href="tl/4/1x4_2.txt" download="1x4_2.txt">1x4_2</a> <br> <a href="/GroundForge/sheet.html?{{ "patch=5632;checker" }}">SVG</a>
+<a href="tl/4/2x2_3.txt" download="2x2_3.txt">2x2_3</a> | <a href="tl/4/1x1_1.txt" download="1x1_1.txt">1x1_1</a> | <a href="tl/4/1x2_1.txt" download="1x2_1.txt">1x2_1</a>  | <a href="tl/4/1x3_1.txt" download="1x3_1.txt">1x3_1</a> | <a href="tl/4/1x4_2.txt" download="1x4_2.txt">1x4_2</a>
```
Comment (jo-pol) [inline docs/index.md:43]: Preview result: 
image

Comment (jo-pol) [reply docs/index.md:43]: Oh, first on this page: 
image

Comment (jo-pol) [reply docs/index.md:43]: Did you know you can create download links without files?

```html
<a download="blabla.txt" href="data:text/plain,CHECKER%094%094%0D[0,0,-1,1,0,2]%09[2,0,2,2,3,1]%09%0D[3,1,2,2,4,2]%09%0D[0,2,0,4,1,3]%09[2,2,1,3,2,4]%09%0D[1,3,0,4,2,4]">txt</a>
```

perhaps I don't need to tell you: %09 is a tab, %0D a new line

**Comment by veronika** [comment]
I have moved the "See also" links out of the table.
I have also provided an SVG link for each ground.  It looks very similar to the original now.

**Pull request closed without merging**